pages/md-style-guide.md

```diff
@@ -1,145 +1,230 @@
----
```

Comment:
Why was all of this content replaced with what looks like primarily CSS?  Where is this content now?